modules/Core/queries/users.php

```diff
@@ -1,18 +1,20 @@
 <?php
 // Searchable user list
-if (!$user->isLoggedIn()) {
-    die(json_encode(['error' => 'Unauthenticated']));
-}
-
-if (!isset($_GET['search']) || strlen($_GET['search']) < 3) {
-    die(json_encode(['error' => 'Please enter a search query of at least 3 characters']));
+if (!isset($_GET['search']) || strlen($_GET['search']) < 2) {
```

Comment:
I wonder if we should look at rate limiting this query

Reply:
Added - 60 rq/m.
Also added a default `limit 5` clause to the sql query.